roles/certbot-cloudflare/README.md

```diff
@@ -36,12 +36,14 @@ systemctl status certbot
 
 # Variables in this Role
 
-| variable             | example              | purpose                                                                    |
-|:---------------------|:---------------------|:---------------------------------------------------------------------------|
-| admin_username       | admin                | who you log into the server as, instead of root.                           |
-| domain               | ansible.fun          | fqdn of host server. should be set in inventory                            |
-| wildcard_needed      | true                 | whether role should get certs for *.domain too.                            |
-| cloudflare_api_token | someApiString        | the zoned token created in cloudflare.                                     |
-| cert_email           | ops@planetary.social | email used when assigning cert, for receiving notices (renewal,issues,etc) |
+| variable             | example                    | purpose                                                                    |
+|:---------------------|:---------------------------|:---------------------------------------------------------------------------|
+| admin_username       | admin                      | who you log into the server as, instead of root.                           |
+| domain               | ansible.fun                | fqdn of host server. should be set in inventory                            |
+| wildcard_needed      | true                       | whether role should get certs for *.domain too.                            |
+| cloudflare_api_token | someApiString              | the zoned token created in cloudflare.                                     |
+| cert_email           | ops@planetary.social       | email used when assigning cert, for receiving notices (renewal,issues,etc) |
+| homedir              | /home/{{ admin_username }} | the home directory of the current user                                     |
```

Comment:
The diff got bigger here but I just added the `homedir` var and adjusted the columns to fit.

Reply:
Ah, this readme needed to be updated.  We no longer need the `wildcard_needed` var.  Instead, this role looks for a domain var and, if it starts with `*.` will automatically make a wildcard_domain.  If it doesn't find a domain var, it will use the inventory hostname instead.  I updated the readme([9687577](https://github.com/planetary-social/ansible-scripts/pull/48/commits/96875771421897d6cddbfd9d8f72ce1755a0d32f)) and added it to this pr.

Reply:
oh that's a nice update.